From c10824a0520daddf94535bd3848136b240b07c3c Mon Sep 17 00:00:00 2001
Message-ID: <c10824a0520daddf94535bd3848136b240b07c3c.1784292333.git.mjg@fedoraproject.org>
From: Michael J Gruber <mjg@fedoraproject.org>
Date: Thu, 6 Jun 2024 14:27:12 +0200
Subject: [PATCH] Do not complain to your friendly local distribution manager

Artifex decided to rely on local forks for added functionality, so do
not complain to the distribution manager about lack of functionality in
unbundled builds.
---
 platform/gl/gl-ui.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/platform/gl/gl-ui.c b/platform/gl/gl-ui.c
index c0cf69515..c0a71fa58 100644
--- a/platform/gl/gl-ui.c
+++ b/platform/gl/gl-ui.c
@@ -452,8 +452,6 @@ void ui_init(int w, int h, const char *title)
 #if defined(FREEGLUT) && (GLUT_API_VERSION >= 6)
 	glutKeyboardExtFunc(on_keyboard);
 #else
-	fz_warn(ctx, "This version of MuPDF has been built WITHOUT clipboard or unicode input support!");
-	fz_warn(ctx, "Please file a complaint with your friendly local distribution manager.");
 	glutKeyboardFunc(on_keyboard);
 #endif
 	glutSpecialFunc(on_special);
-- 
2.55.0.417.gd2046ead17

